components/2.0/PasswordConnections.qml

```diff
@@ -0,0 +1,151 @@
+/***************************************************************************
```

Comment:
I'm not super convinced by some of this class. 
We're creating a template style approach (which could be fine), but only for one minor feature. 

it's just as easy to add:

enabled: !renewalDialog.visible into the theme in the right places.
as it is to pass a list of items in here.

Reply:
sure, we can also set enabled in the themes to block the item(s). the disabledItems was though just as a helper to collect/bundle the logic for password renewal. i can kick it out if you dont like it.